openBCIBoard.js

```diff
@@ -1788,49 +1824,124 @@ function OpenBCIFactory() {
 
     /**
      * @description A method to parse a stream packet that does not have channel data or aux/accel data, just a timestamp
-     * @param rawPacket - A 33byte data buffer from _processQualifiedPacket
+     * @param rawPacket {Buffer} - A 33byte data buffer from _processQualifiedPacket
+     * @param timeOfPacketArrival {Number} - The time the packet arrived.
      * @private
+     * @returns {Promise}
      * @author AJ Keller (@pushtheworldllc)
      */
-    OpenBCIBoard.prototype._processPacketTimeSyncSet = function(rawPacket) {
+    OpenBCIBoard.prototype._processPacketTimeSyncSet = function(rawPacket, timeOfPacketArrival) {
         return new Promise((resolve, reject) => {
-            var currentTime = this.sntpNow();
+            if (this.sync.curSyncObj === null) reject('no sync in progress');
+            // console.log('hey');
+            this.sync.curSyncObj.timeSyncSetPacket = timeOfPacketArrival;
+
             if (this.options.verbose) console.log('Got time set packet from the board');
+
+            // this.curParsingMode will equal k.OBCIParsingNormal if comma found
```

Comment:
`<` is sent to dongle and dongle sends `,` back to driver when it actually sends out that `<` to the Board